tracer/test/Datadog.Trace.Tests/LambdaCommonTests.cs

```diff
@@ -19,27 +19,30 @@ namespace Datadog.Trace.Tests
 {
     public class LambdaCommonTests
     {
-        private readonly Mock<ILambdaExtensionRequest> _lambdaRequestMock = new Mock<ILambdaExtensionRequest>();
+        private readonly Mock<ILambdaExtensionRequest> _lambdaRequestMock = new();
 
         [Fact]
         public void TestCreatePlaceholderScopeSuccessWithTraceIdOnly()
         {
             var tracer = TracerHelper.Create();
             var scope = LambdaCommon.CreatePlaceholderScope(tracer, "1234", null);
+
             scope.Should().NotBeNull();
-            scope.Span.TraceId.ToString().Should().Be("1234");
-            scope.Span.SpanId.ToString().Should().NotBeNull();
-            scope.Span.Context.TraceContext.SamplingPriority.Should().Be(1);
+            scope.Span.TraceId128.Should().Be((TraceId)1234);
+            scope.Span.TraceId.Should().Be(1234);
+            scope.Span.SpanId.Should().BeGreaterThan(0);
```

Comment:
I think you may have accidentally deleted a line:

```suggestion
            scope.Span.SpanId.Should().BeGreaterThan(0);
            scope.Span.Context.TraceContext.SamplingPriority.Should().Be(1);
```

Reply:
Unrelated to 128-bit trace ids, but I intentionally removed the check for sampling priority. There should not be an expectation here that it will _always_ be `1` unless we set an explicit `ITraceSampler` or sampling rules.

There are other tests in this file specifically for different values of the sampling priority string.